docker-compose.yml

```diff
@@ -53,6 +55,7 @@ services:
       - elasticsearch
 
   kibana:
+    container_name: kibana
```

Comment:
I assume these names are necessary to be able to call services internally between each other. 
We can rely on the implicit docker-compose names for the containers. If we need explicit container names, lets set them only for the services that rely on the custom names.